src/libraries/System.Private.CoreLib/src/System/Diagnostics/StackTrace.cs

```diff
@@ -322,6 +323,16 @@ internal void ToString(TraceFormat traceFormat, StringBuilder sb)
                             sb.Append(' ');
                             sb.AppendFormat(CultureInfo.InvariantCulture, inFileLineNum, fileName, sf.GetFileLineNumber());
                         }
+                        else
+                        {
+                            if (!string.IsNullOrEmpty(System.Environment.GetEnvironmentVariable("COMPlus_PrintILOffset")))
```

Comment:
The bool should be cached in a static rather than being checked for every line of every exception?

Reply:
Thanks. I'll fix it.
Additionally, I'll change envrionment name from "COMPlus_PrintILOffset" to "COMPlus_ILOffsetToStacktrace"

Reply:
How about changing to ```COMPlus_ILOffsetToStackTrace```?

Reply:
"COMPlus_ILOffsetToStackTrace" looks more proper. I'll fix